plugins/bcm57xx/fu-bcm57xx-recovery-device.c

```diff
@@ -90,7 +90,7 @@ typedef union {
 		guint32 Req2 : 1;
 		guint32 Req3 : 1;
 		guint32 reserved_31_16 : 16;
-	} __attribute__((packed)) bits;
```

Comment:
Shouldn't these be changed to rustgen?

Reply:
Ya, I'm working my way through these (FuCrosEcUsbDevice is the plan for today) -- although in this case I was thinking of _removing_ `fu-bcm57xx-recovery-device.c` -- it abuses /dev/mem and will only work with secure boot turned off. I think it's better to get this PR in quickly and then work through the old ones async -- as there are so many more plugins in the work I'm trying to optimize "lets review the plugin upstream" task.

Reply:
I'll move FuCrosEcUsbDevice out to a different PR as I want to ask Benson some questions.